src/wallet-core/wallet-core.ts

```diff
@@ -56,7 +56,7 @@ export interface IAccount {
 }
 
 export class WalletCore {
-  accounts: Array<IAccount>;
+  accounts: Array<AntennaAccount>;
```

Comment:
if we need to expose new interfaces, just add them to the IAccount. We'd better not expose the implementation layer classes.